easybuild/easyconfigs/r/robin-map/robin-map-1.4.0-GCCcore-13.3.0.eb

```
@@ -0,0 +1,26 @@
+# Authors:: Tanmoy Chakraborty (University of Warwick/tanmoy.chakraborty@warwick.ac.uk)
+
+easyblock = 'CMakeMake'
+
+name = 'robin-map'
+version = '1.4.0'
+
+homepage = 'https://github.com/Tessil/robin-map'
+description = """robin-map is a C++ implementation of a fast and memory efficient hash table.
+It is based on Robin Hood hashing with backward shift deletion."""
+
+source_urls = ['https://github.com/Tessil/robin-map/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['7930dbf9634acfc02686d87f615c0f4f33135948130b8922331c16d90a03250c']
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+dependencies = [('CMake', '3.29.3')]
+builddependencies = [('binutils', '2.42')]
+
```

```suggestion
toolchain = {'name': 'GCCcore', 'version': '13.3.0'}

source_urls = ['https://github.com/Tessil/robin-map/archive/refs/tags/']
sources = ['v%(version)s.tar.gz']
checksums = ['7930dbf9634acfc02686d87f615c0f4f33135948130b8922331c16d90a03250c']

builddependencies = [
    ('binutils', '2.42'),
    ('CMake', '3.29.3'),
]

```

cmake is of course a build dep. Lets also keep the order toolchain -> sources + checksums -> builddeps -> deps